kevm-pyk/src/kevm_pyk/gst_to_kore.py

```diff
@@ -113,7 +134,8 @@ def kore_pgm_to_kore(pgm: Pattern, pattern_sort: SortApp, schedule: str, mode: s
 
 
 def _schedule_to_kore(schedule: str) -> App:
-    return App(f"Lbl{schedule}'Unds'EVM")
+    escaped_schedule = schedule.replace('_', "'Unds'")
+    return App(f"Lbl{escaped_schedule}'Unds'EVM")
```

Comment:
This should be fine, but a potentially more robust solution is to use `munge` (`from pyk.konvert import munge`: https://github.com/runtimeverification/k/blob/e064582a5a4e0f00bc8c32bee86f4d1382887ea2/pyk/src/pyk/konvert/_utils.py#L53)

```suggestion
    return App(f"Lbl{munge(schedule)}'Unds'EVM")
```